src/media-groups.js

```diff
@@ -105,6 +105,17 @@ export const onGroupChanged = (type, settings) => () => {
   startLoaders(activeGroup.playlistLoader, mediaType);
 };
 
+const onGroupChanging = (type, settings) => () => {
+  const {
```

Comment:
Should this use stopLoaders? And should onGroupChanged no longer stopLoaders?

Reply:
Originally I tried using `stopLoaders`  but that caused an issue because we abort the current audio playlist loader request when starting playback even though we aren't changing audio playlists. I will check if `mediachange` should just stop the audio playlist loader, since the segmentLoader will be stopped.

Reply:
OK after testing out changing `onGroupChanged` to only stop the `activePlaylistLoader` is doesn't work. Loaders are started and stopped frequently during the start of playback and if we don't stop them in `mediachange` too it can cause playback to hang because the segment loader is trying to load, but there isn't an `activePlaylistlistLoader`.

Reply:
test added